packages/artillery-plugin-publish-metrics/lib/dynatrace/index.js

```diff
@@ -5,6 +5,12 @@ const debug = require('debug')('plugin:publish-metrics:dynatrace');
 
 class DynatraceReporter {
   constructor(config, events, script) {
+    if (!config.apiToken || !config.envUrl) {
+      throw new Error(
+        'Dynatrace API Access Token or Environment URL not specified. In order to send metrics to Dynatrace both `apiToken` and `envUrl` must be set'
```

Comment:
Suggestion: use actual property names as the user would type them in. In this instance something like:

```
Dynatrace reporter: both apiToken and envUrl must be set.
```

We could also link to the docs using `docs.art` short URL  (https://docs.art/reference/extensions/publish-metrics#dynatrace) but would need to see what the formatting would look like in the terminal.

Reply:
Took some screenshots of how it looks like. First one is with the extra empty line to make it more visible, and the second without the additional `\n`. 
Let me know what you think @hassy.

<img width="1172" alt="Screenshot 2023-08-14 at 21 59 26" src="https://github.com/artilleryio/artillery/assets/39635558/96af7e1b-dcf0-48a0-8128-ef189b5ac06c">


.

<img width="1162" alt="Screenshot 2023-08-14 at 21 59 49" src="https://github.com/artilleryio/artillery/assets/39635558/458a4095-1ae1-479c-b07c-7fbcd25687a5">

Reply:
We should avoid new lines when creating and throwing `Error` instances I think - formatting should be handled in a different place.

Reply:
Noted 🙌🏻  Otherwise using the link in the error message looks ok to you @hassy ? Can change all of them accordingly?